api/app/graphql/mutations/send_otp.rb

```diff
@@ -0,0 +1,25 @@
+module Mutations
+  class SendOtp < BaseMutation
+
+    require_relative '../../../lib/app_messaging_service.rb'
```

Comment:
since the `lib` folder is inside the bundlers load path, it is not needed in this file.